src/components/pages/Map.tsx

```diff
@@ -137,16 +140,29 @@ function Map() {
           onBoundsChanged={onMapBoundsChanged}
           zoom={13}
         >
-          {/* <Marker position={positionTokyo} />
-          <InfoWindow position={positionTokyo} options={infoWindowOptions}>
-            <div style={divStyle}>
-              <h1>ガンダムベース東京</h1>
-            </div>
-          </InfoWindow> */}
+          
+          {/* Railsから取得したデータを、Marker地図上に表示 */}
+          {shops.map((shop, index) => (
+            <>
+              {/* <Marker position={shop} />
+              <InfoWindow position={shop} options={infoWindowOptions}>
+                <div style={divStyle}>
+                  <h1>{shop.name}</h1>
+                </div>
+              </InfoWindow> */}
+              <Marker position={{ lat: 35 + index, lng: 139 + index }} key={`marker-${index}`} />
+              <InfoWindow position={{ lat: 35 + index, lng: 139 + index }} options={infoWindowOptions} key={`info-${index}`}>
```

Comment:
こんな感じにしたら表示されました！
APIから返ってくるlatとlngが文字列になっていたので数値に変換してます！
```suggestion
              <Marker position={{ lat: Number(shop.lat), lng: Number(shop.lng) }} key={`marker-${index}`} />
              <InfoWindow position={{ lat: Number(shop.lat), lng: Number(shop.lng) }} options={infoWindowOptions} key={`info-${index}`}>
```
![スクリーンショット 2022-06-22 21 53 12](https://user-images.githubusercontent.com/18608862/175033816-18e6d3ac-4d99-4c89-901e-871d95663fe6.png)